.github/workflows/build-base-images.yml

```diff
@@ -32,7 +32,7 @@ jobs:
       - name: Output Variables
         id: vars
         run: |
-          docker_md5=$(find . -type f \( -name "build/Dockerfile" -o -name .github/data/version.txt \) -not -path "./docs*"  -exec md5sum {} + | LC_ALL=C sort  | md5sum | awk '{ print $1 }' )
+          docker_md5=$(find . -type f \( -name "Dockerfile" -o -name version.txt \) -not -path "./tests*" -exec md5sum {} + | LC_ALL=C sort  | md5sum | awk '{ print $1 }')
```

Comment:
we can also exclude `/perf-tests`

Reply:
Is there a Dockerfile in `/perf-tests`?  This is what this filter is for.

Reply:
my bad I thought it was excluding dir